{{cookiecutter.project_slug}}/.neuro/live.yml

```diff
@@ -88,46 +83,23 @@ jobs:
         python $[[ volumes.code.mount ]]/train.py --data $[[ volumes.data.mount ]] $[[ multi.args ]]
 
   jupyter:
-    multi: True
-    image: $[[ images.myimage.ref ]]
-    http_port: 8888
-    http_auth: True
-    browse: True
-    detach: True
-    volumes:
-      - $[[ volumes.data.ref_ro ]]
-      - $[[ volumes.code.ref_rw ]]
-      - $[[ volumes.config.ref_ro ]]
-      - $[[ volumes.notebooks.ref_rw ]]
-      - $[[ volumes.results.ref_rw ]]
-    env:
-      PYTHONPATH: $[[ volumes.code.mount ]]
-    cmd: >-
-      jupyter $[[ multi.args ]]
-        --no-browser
-        --ip=0.0.0.0
-        --allow-root
-        --NotebookApp.token=
-        --notebook-dir=$[[ volumes.notebooks.mount ]]
+    action: gh:neuro-actions/jupyter@master
```

Comment:
where is the action's repo located? https://github.com/neuro-inc/neuro-actions/jupyter -> 404

Reply:
We've got a separate GH org for neuro-flow actions: https://github.com/neuro-actions/jupyter